packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -53,16 +53,22 @@ describe("Garbage Collection Tests", () => {
     // Update this for individual node to update the initial GC state of that node.
     const emptyGCDetails: IGarbageCollectionSummaryDetails = {};
 
+    // The flag for the close call
+    let closeCalled = false;
+
     const createGarbageCollector = (
         baseSnapshot: ISnapshotTree | undefined = undefined,
         getNodeGCDetails: (id: string) => IGarbageCollectionSummaryDetails = () => emptyGCDetails,
     ) => {
         mockLogger = new MockLogger();
         return GarbageCollector.create(
             gcRuntime,
-            { gcAllowed: true, deleteTimeoutMs },
+            { gcAllowed: true, deleteTimeoutMs, gcTestSessionTimeoutMs: 0 },
```

Comment:
You shouldn't do this here. This will expiry the session immediately for all tests. It doesn't really matter currently because nothing really happens when garbage collector is disposed. However, that may change and tests will start failing. You can accept this as an optional parameter to createGarbageCollector and only pass it in the session expiry tests.